geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/CommandInitializer.java

```diff
@@ -338,7 +338,9 @@ public static void registerCommand(int messageType, Map<Version, Command> versio
     allCommands.put(Version.GEODE_1_10_0, geode18Commands);
     allCommands.put(Version.GEODE_1_11_0, geode18Commands);
     allCommands.put(Version.GEODE_1_12_0, geode18Commands);
+    allCommands.put(Version.GEODE_1_12_1, geode18Commands);
     allCommands.put(Version.GEODE_1_13_0, geode18Commands);
+    allCommands.put(Version.GEODE_1_13_1, geode18Commands);
```

Comment:
Will need another PR to add GEODE_1_12_1 and GEODE_1_13_1 to develop as well

Reply:
😭